runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineConfiguration.java

```diff
@@ -146,6 +149,11 @@ public URL configURL()
         return ENGINE_CONFIG_URL.get(this);
     }
 
+    public Path configPath()
+    {
+        return ENGINE_CONFIG_PATH.get(this);
+    }
+
```

Comment:
This `configPath()` method looks good.

Let's keep the `ENGINE_CONFIG_URL` constant in place for now, to support `zilla.properties` defining `zilla.engine.config.url` property, but remove all calls to `EngineConfiguration.configURL()` from the code, and replace with corresponding `EngineConfiguration.configPath()` method calls instead.